class/router.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const router = express.Router();
+const jsonParser = bodyParser.json();
+
+
+router.get('/', (req, res) => {
+    res.json([{'classname': 'BIO103'}]);
+  });
+
+
+router.post('/', jsonParser, (req, res) => {
+   
+    const item = {'id':'123', 'firstName':'John', 'lastName':'Doe', 'username':'username', 'className':'BIO103'}
```

Comment:
Can this be the multiline object form? All on one line is hard to read for 14.

Reply:
Okay, should be all set. If cool, I'm going to continue to build out the endpoints to accept basic data from the db.